Assets/Scripts/SceneManageMent/ScoreTracker.cs

```
@@ -130,55 +125,31 @@ public void AddToScore(int points)
         currentScore += points;
     }
 
-    /// <summary>Updates this class's Energy to the bike's energy.</summary>
-    private void UpdateUIEnergy()
+    /// <summary>
+    /// Calls ResetGameObject() on every IResettable object in the game world
+    /// </summary>
+    private void GameReset()
     {
-        if (bike == null)
-        {
-            Debug.Log("There is no bike.");
```

This should probably throw an exception and kill the game, what would happen if we try to play the game with no bike?